main.go

```diff
@@ -1,55 +1,27 @@
 package main
 
 import (
-	"context"
 	"fmt"
 	"log"
 	"os"
-	"strings"
-
-	"github.com/ikawaha/slackbot/socketmode"
 )
 
 func main() {
-	if len(os.Args) != 4 {
-		fmt.Fprintf(os.Stderr, "usage: bot app-level-token slack-bot-token name")
+	if len(os.Args) != 3 {
+		fmt.Fprintf(os.Stderr, "usage: bot app-level-token slack-bot-token")
 		os.Exit(1)
 	}
-
-	// start a websocket-based Real Time API session
-	bot, err := NewBot(os.Args[1], os.Args[2], os.Args[3])
+	debug := false
+	bot, err := NewBot(os.Args[1], os.Args[2], debug)
 	if err != nil {
 		log.Fatal(err)
+		panic(err)
 	}
-	defer bot.Close()
 	fmt.Println("^C exits")
 
-	mentionTag := "<@" + bot.ID + ">"
-	for {
-		if err := bot.ReceiveMessage(context.TODO(), func(ctx context.Context, ev *socketmode.Event) error {
-			log.Printf("bot_id: %v, msg_user_id: %v, event: %+v\n", bot.ID, ev.UserID, ev)
-			if ev.IsSlashCommand() {
-				dictType := IPA
-				if strings.HasSuffix(ev.Command, string(NEOLOGD)) {
-					dictType = NEOLOGD
-				} else if strings.HasSuffix(ev.Command, string(UNI)) {
-					dictType = UNI
-				}
-				command := fmt.Sprintf("%v %v", ev.Command, ev.Text)
-				if err := bot.PostMessage(context.TODO(), ev.Channel, command); err != nil {
-					log.Printf("post message failed, msg: %+v, %v", ev, err)
-				}
-				go bot.ResponseWithDictType(ev, dictType)
-				return nil
-			}
-			if ev.IsMessage() && strings.HasPrefix(ev.Text, mentionTag) {
-				ev.Text = strings.TrimSpace(ev.Text[len(mentionTag):])
-				go bot.Response(ev)
-				return nil
-			}
-			return nil
-		}); err != nil {
-			log.Fatal(err)
-		}
+	err = bot.Run()
+	if err != nil {
+		log.Fatal(err)
+		panic(err)
```

Comment:
```suggestion
		log.Fatal(err)
```
see. https://cs.opensource.google/go/go/+/refs/tags/go1.21.6:src/log/log.go;l=410-413